airflow/operators/docker_operator.py

```diff
@@ -198,7 +198,7 @@ def execute(self, context):
 
         if self.force_pull or len(self.cli.images(name=self.image)) == 0:
```

Comment:
```suggestion
         if self.force_pull:
            ignore = ["Downloading", "Extracting", "Waiting"]
            self.log.info('Force pulling docker image %s', self.image)
            for output in self.cli.pull(self.image, stream=True, decode=True):
                if 'status' in output and not any(w in output['status'] for w in ignore):
                    self.log.info("%s", output['status'])
```

Reply:
This part is not from me, but I will look into it too. Thanks!